repairnator/repairnator-core/src/main/java/fr/inria/spirals/repairnator/LauncherUtils.java

```diff
@@ -284,11 +284,9 @@ public static void checkArguments(JSAP jsap, JSAPResult arguments, LauncherType
     }
 
     public static void checkEnvironmentVariable(String envVariable, JSAP jsap, LauncherType launcherType) {
-        if (launcherType != LauncherType.PIPELINE) {
-            if (System.getenv(envVariable) == null || System.getenv(envVariable).equals("")) {
-                System.err.println("You must set the following environment variable: "+envVariable);
-                LauncherUtils.printUsage(jsap, launcherType);
-            }
+        if (System.getenv(envVariable) == null || System.getenv(envVariable).equals("")) {
```

Comment:
This won't work: this method is used also to check GITHUB_OAUTH and the env variable is not available when running the pipeline through the docker image: we use the Github token through an argument (the idea behind it is to avoid keeping Github token as an env variable to avoid leaking it in case of env variable dumping in logs).

In short, to avoid any confusion here, we should have two specific methods: one to check Github Oauth and the other to check M2_HOME.

Reply:
Oh, ok. I will fix that.

Reply:
I tried a different approach without creating two separate methods. I liked this approach because `checkEnvironmentVariable` is only called from `checkArguments` now. See what you think. If you still prefer two separate methods, I will change that.

Reply:
I'm not a huge fan but it should work. Let's keep it for now.